### PR TITLE
Consolidate *loadOp and clear* into a unioned field

### PR DESCRIPTION
**Pull request by kainino0x**
Followup to #276.

***
<a href="https://pr-preview.s3.amazonaws.com/kainino0x/gpuweb/pull/283.html" title="Last updated on Jul 22, 2019, 9:26 PM UTC (488987e)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/gpuweb/gpuweb/283/c6db522...kainino0x:488987e.html" title="Last updated on Jul 22, 2019, 9:26 PM UTC (488987e)">Diff</a>

### CODE REVIEW

File: spec/index.bs [hunk 1]
```diff
@@ -620,8 +620,9 @@ interface GPUComputePassEncoder : GPUProgrammablePassEncoder {
 
 
 enum GPULoadOp {
-    "clear",
-    "load"
+    "load",
+    "opaqueBlack",
+    "zero"
```
Comment (kainino0x) [inline spec/index.bs]: opaqueBlack and zero are up for discussion.

Comment (jdashg) [reply spec/index.bs]: Prefer not to duplicate functionality of GPUColor.

**Comment by kvark** [comment]
@jdashg 
> TBH I think I prefer using a nullable clearColor, such that "load" is the default, but we'll clear instead if users specify a clearColor.

That's exactly what #276 did, and your feedback was that it was "gross".
The reasoning was discussed in [the call](https://docs.google.com/document/d/1Lihb8Az6_xaizjAIQsu8cg0IEDBFOChTZGusd9D_ABE/edit#heading=h.ur4f2isiy2xk) as well as in https://github.com/gpuweb/gpuweb/pull/276#issuecomment-487195089.

TL;DR: defaulting to non-clear is bad for performance

**Comment by jdashg** [comment]
My opinions are mutable, and my comment was made for my /Approval/ of that PR!

My preferences:
1) loadOp: "clear", "load"
    clearColor with default
2) clearColor nullable, null => load
3) This PR

**Comment by kvark** [comment]
@jdashg understood. Please also check out #284

**Comment by kainino0x** [comment]
Merging with verbal approval in the meeting from Myles, Rafael, and others.
